src/Extensions/ElementalPageExtension.php

```diff
@@ -37,4 +38,25 @@ public function MetaTags(&$tags)
             $tags = $html->getContent();
         }
     }
+
+    public function updateCMSFields(FieldList $fields)
+    {
+        $fields = parent::updateCMSFields($fields);
+        $gf = $fields->dataFieldByName('ElementalArea');
+        $cfg = $gf->getConfig();
+        $cfg->removeComponentsByType([
+            '\SilverStripe\Forms\GridField\GridFieldEditButton',
+            '\SilverStripe\Forms\GridField\GridFieldDeleteAction',
+            // Detail form will be added back, but a lower priority order for route processing
+            // We need MeatballMenu to handle the same routes. Detail form is required by
+            // \Symbiote\GridFieldExtensions\GridFieldAddNewMultiClass so can't just be removed,
+            // since it AddNewMultiClass is also utilised by Elemental.
+            '\SilverStripe\Forms\GridField\GridFieldDetailForm'
+        ]);
+        $cfg->addComponents(
+            new \Symbiote\GridFieldExtensions\GridFieldMeatballMenuComponent,
+            new \SilverStripe\Forms\GridField\GridFieldDetailForm
+        );
+        return $fields;
```

Comment:
Some initial comments:

* We should add these components to `ElementalEditor` where the GridField is first initialised
* Can we please refer to class names using `ClassName::class` and namespace imports rather than FCQN strings?
* Ideally use `Injector::inst()->create()` rather than creating new classes directly